dbms/src/Parsers/IAST.h

```diff
@@ -45,6 +42,18 @@ class IAST : public std::enable_shared_from_this<IAST>
     IAST(const IAST &) = default;
     IAST & operator=(const IAST &) = default;
 
+    template <class Derived>
+    Derived * As()
```

Comment:
Why it's in upper case?

Reply:
Good question. I will rename to lower case.